packages/augur-state/src/db/TrackedUsers.test.ts

```diff
@@ -0,0 +1,25 @@
+import { TrackedUsers } from "./TrackedUsers";
+import { AbstractDB } from "./AbstractDB";
+
+
+async function wipeDb(DbClass: new () => AbstractDB) {
```

Comment:
I heavily changed how database setup works. Instead of subclassing to get access to protected values, I use the element access syntax `["db"]`.